content/en/logs/guide/how-to-set-up-only-logs.md

```diff
@@ -31,7 +38,7 @@ To disable payloads, you must be running Agent v6.4+. This disables metric data
 {{% /tab %}}
 {{% tab "Docker" %}}
 
-If you are using the Docker containerized Agent, set the `DD_ENABLE_PAYLOADS_EVENTS`, `DD_ENABLE_PAYLOADS_SERIES`, `DD_ENABLE_PAYLOADS_SERVICE_CHECKS`, and `DD_ENABLE_PAYLOADS_SKETCHES` environment variables to `false` in addition to your Agent configuration:
+If you're using the Docker containerized Agent, make sure to set the following environment variables in addition to your Agent configuration:
```

Comment:
The sentence says "the following environment variables", but what follows is a docker command which is only partially relevant to the described scenario, making it more difficult to see which variables need to be set to false. Would it make sense to keep the list of variables to set, or adjust this section to be more like the Kubernetes one?

Reply:
Thanks for clarifying, I'll break out the instructions for clarity!

```suggestion
If you're using the Docker containerized Agent, set the following environment variables to `false`:
- `DD_ENABLE_PAYLOADS_EVENTS`
- `DD_ENABLE_PAYLOADS_SERIES`
- `DD_ENABLE_PAYLOADS_SERVICE_CHECKS`
- `DD_ENABLE_PAYLOADS_SKETCHES`

Here's an example of how you can include these settings in your Docker run command:
```